libs/pages/application/src/lib/ui/container/container.tsx

```diff
@@ -124,40 +125,39 @@ export function Container(props: PropsWithChildren<ContainerProps>) {
   )
 
   const headerActions = (
-    <>
+    <div className="flex flex-col gap-3">
+      <div className="flex flex-row gap-2">
+        {environment && (
+          <Skeleton width={80} height={32} show={!environment?.mode}>
+            <EnvironmentMode size="xs" mode={environment.mode} />
+          </Skeleton>
+        )}
+        <Skeleton width={40} height={32} show={!environment?.cloud_provider}>
+          <Badge size="xs" variant="outline">
```

Comment:
You can remove this badge, not necessary

Reply:
https://github.com/Qovery/console/pull/850/commits/33af24ea5d4dc5430202d89640c36ffdfcc6467c